lib/rucio/client/replicaclient.py

```diff
@@ -49,7 +49,7 @@ def declare_bad_file_replicas(self, pfns, reason):
         """
         Declare a list of bad replicas.
 
-        :param pfns: The list of PFNs.
```

Comment:
What's the difference of this to the method below?

Reply:
Before it was only a list of PFNs (string), now it can be either a list of PFNs (string) or a list of replicas {'scope': <scope>, 'name': <name>, 'rse_id': <rse_id>}
Or did I misunderstand your question ?

Reply:
It's not exactly the same, but quite similar to the `declare_bad_did_replicas` below. I am just wondering, can we remove that one then?

Reply:
`declare_bad_did_replicas` was introduced for CMS AFAIR. @ericvaandering should comment